sc/src/main/java/uk/ac/ebi/atlas/download/FileDownloadController.java

```diff
@@ -91,4 +92,42 @@ public FileDownloadController(ExperimentFileLocationService experimentFileLocati
 
         zipOutputStream.close();
     }
+
+    @RequestMapping(value = "experimentlist/download/zip",
```

Comment:
We agreed on using hyphens for multiword URLs. Rename the path to `experiment-list/...`. Also, not really sure if `list` serves any purpose here. I’d keep it simple.

Reply:
For now, I have changed to `experiments/download/zip` to keep it simple.